src/testIntegration/kotlin/uk/gov/justice/digital/hmpps/sendlegalmailtoprisonsapi/E2eTest.kt

```diff
@@ -59,7 +59,7 @@ class E2eTest(
       .retrieve()
       .bodyToMono(Message::class.java)
       .block() as Message
-    val (secretValue) = ".*secret=(.*)$".toRegex().find(message.source)!!.destructured
+    val (secretValue) = ".*secret=(.*)".toRegex().find(message.source)!!.destructured
```

Comment:
Now that the email body has changed, the secret is no longer the end of the content (ie. the `$` in the regex). Removed the `$` end of string anchor and the test passes again 👍